tool/tsh/tsh.go

```diff
@@ -1987,7 +2030,8 @@ func printNodesWithClusters(nodes []nodeListing, verbose bool) {
 	} else {
 		t = asciitable.MakeTableWithTruncatedColumn([]string{"Proxy", "Cluster", "Node Name", "Address", "Labels"}, rows, "Labels")
 	}
-	fmt.Println(t.AsBuffer().String())
+
+	t.AsBuffer().WriteTo(os.Stdout)
```

Comment:
Was removing the last printed newline from `fmt.Println` intended here? It's also writing bytes instead of converting to string first, but it should be ascii so I think that's fine?